ovirt/resource_ovirt_vm.go

```diff
@@ -566,6 +579,41 @@ func resourceOvirtVMCreate(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
+	isAutoPinning := false
+	// Mimic the UI behavior. unless specified set to existing policy for high performance VMs.
+	if _, ok := d.GetOk("auto_pinning_policy"); !ok && isHighPerformance {
```

Comment:
I think we should return an error in case the user set auto pining and no high performance since currently the auto pining will just not happen and the user will not know that